packages/ts-morph/src/tests/issues/1494tests.ts

```diff
@@ -0,0 +1,42 @@
+import { ts } from "@ts-morph/common";
+import { expect } from "chai";
+import { Project } from "../../Project";
+
+describe("tests for my issue", () => {
+  it("should update specifiers with path aliases during move", () => {
+    const project = new Project({
+      useInMemoryFileSystem: true,
+      compilerOptions: {
+        baseUrl: '.',
+        paths: {
+          'Root/*': ['./*']
```

Comment:
If we want to move ahead with this PR we should add a lot more tests for paths.

- With and without wildcard value
- More nested mappings